pkg/runtime/helper.go

```diff
@@ -169,3 +169,13 @@ func (m MultiObjectTyper) IsUnversioned(obj Object) (bool, bool) {
 	}
 	return false, false
 }
+
+// SetZeroValue would set the object of objPtr to zero value of its type.
```

Comment:
This is not possible because of import loop:
```

	imports k8s.io/kubernetes/pkg/api/unversioned
	imports k8s.io/kubernetes/pkg/conversion
	imports k8s.io/kubernetes/pkg/runtime
	imports k8s.io/kubernetes/pkg/api/unversioned
```